app/src/main/java/com/example/farmeraid/data/FarmRepository.kt

```diff
@@ -97,4 +101,16 @@ class FarmRepository(
             }
         } ?: ResponseModel.FAResponseWithData.Error("User is not part of a farm"))
     }
+
+    //Taken from: https://www.techiedelight.com/generate-a-random-alphanumeric-string-in-kotlin/
+    fun generateFarmCode(length: Int) : String {
+        val charset = "ABCDEFGHIJKLMNOPQRSTUVWXTZabcdefghiklmnopqrstuvwxyz0123456789"
+        return (1..length)
+            .map { charset.random() }
+            .joinToString("")
+    }
```

Comment:
Good work generating the alphanumeric code

Reply:
:)